st2common/tests/unit/test_jinja_render_crypto_filters.py

```diff
@@ -46,5 +56,22 @@ def test_filter_decrypt_kv(self):
         })
 
         template = '{{st2kv.system.k8 | decrypt_kv}}'
-        actual = env.from_string(template).render(context)
-        self.assertEqual(actual, secret)
+        actual = self.env.from_string(template).render(context)
+        self.assertEqual(actual, self.secret)
+
+    def test_filter_decrypt_kv_with_user_scope_value(self):
```

Comment:
It would also be a good idea to add "end to end" test which verifies that it works for the following scenarios:

1. API user is provided (if action is scheduled via API, API user should be used)
2. User is not provided - it should default to stanley (if execution is scheduled outside of the API context, no user context is available so stanley is used by default)